lms/djangoapps/courseware/courses.py

```diff
@@ -554,9 +555,71 @@ def get_course_assignments(course_key, user, request, include_access=False):
             complete = block_data.get_xblock_field(subsection_key, 'complete', False)
             past_due = not complete and due < now
             assignments.append(_Assignment(
-                subsection_key, title, url, due, contains_gated_content, complete, past_due, assignment_type
+                subsection_key, title, url, due, contains_gated_content, complete, past_due, assignment_type, None
             ))
 
+            # Load all dates for ORA blocks as separate assignments
+            descendents = block_data.get_children(subsection_key)
+            while descendents:
+                descendent = descendents.pop()
+                descendents.extend(block_data.get_children(descendent))
+                if block_data.get_xblock_field(descendent, 'category', None) == 'openassessment':
+                    graded = block_data.get_xblock_field(descendent, 'graded', False)
```

Comment:
Should we check both graded and scored here?

Reply:
Yeah, probably.